### ISSUE TITLE
feat: Include Identifier in PhotoResponse for iOS

### ISSUE THREAD

**Issue by stephan-fischer** [capacitor-community/media#102]
#### Description
The `PhotoResponse` currently lacks an identifier for photos saved to the iOS media library. Adding a unique identifier, such as the `localIdentifier` provided by the `PHAsset` API, would allow developers to reliably reference and manage saved photos. This feature is already indirectly supported on Android via the `uri`, but it is missing on iOS.

#### Proposed Implementation
1. Use the `PHAsset`'s `localIdentifier` when saving a photo to the iOS media library. This value is unique and can be easily retrieved during the saving process.

     Example:
     ```swift
     let creationRequest = PHAssetCreationRequest.forAsset()
     creationRequest.addResource(with: .photo, data: imageData as Data, options: nil)
     let identifier = creationRequest.placeholderForCreatedAsset?.localIdentifier
    ````
2. Extend the PhotoResponse type to include an optional identifier field

      ```typescript
      interface PhotoResponse {
          filePath?: string;
          identifier?: string; // New field for iOS
      }
      ````

#### Use Case
Including a unique identifier is crucial for developers who need to:

- Reference saved photos later for updates, sharing, or deletion.
- Programmatically manage photos in albums.
- Synchronize photo metadata with a backend.

By aligning with Android's uri behavior, this feature would provide consistency across platforms and enhance the plugin's usability in photo management apps.

#### Example Response
After saving a photo on iOS, the PhotoResponse could look like this:
```json
{
  "filePath": "file:///var/mobile/Containers/Data/Application/ABC123/Documents/photo.jpg",
  "identifier": "6A11E9F7-1234-ABCD-5678-EF90ABCDEF12" // iOS localIdentifier
}
````

#### Additional Notes
This feature would ensure parity between iOS and Android and provide a more robust developer experience for applications relying on media management. Thank you for considering this request!

**Comment by nkalupahana** [comment]
Great idea, I should be able to get this done by end of year. This is also a pretty simple feature, so PRs welcome :) definitely a good first issue.

**Comment by nkalupahana** [comment]
Published: v7.0.1.
